### PR TITLE
Multi screen shot support

### PR DESCRIPTION
**Pull request by Awarua-**

### CODE REVIEW

File: models/issue.js [hunk 1]
```diff
@@ -14,8 +14,8 @@ var issueSchema = new mongoose.Schema({
 	histUndoPossible: Boolean,
 	navBackwardPossible: Boolean,
 	navForwardPossible: Boolean,
-	screenshot: String,
+	screenshot: Array,
```
Comment (drkno) [inline models/issue.js:17]: the frontend will need to be fixed

File: rest.js [hunk 2]
```diff
@@ -73,10 +73,9 @@ REST_ROUTER.prototype.handleRoutes= function(router, md5, self) {
                             "description": "If navigation backwards was possible",
                             "type": "Boolean"
                         },
-
-                        "screenshot": {
-                            "description": "An image encoded in base64",
-                            "type": "String"
+                        "screenshots": {
```
Comment (Awarua-) [inline rest.js]: Need to fix that should not be plural

File: rest.js [hunk 3]
```diff
@@ -73,10 +73,9 @@ REST_ROUTER.prototype.handleRoutes= function(router, md5, self) {
...
+                            "description": "A list of images encoded in base64",
+                            "type": "Array"
```
Comment (drkno) [inline rest.js:78]: front end problems

File: updateDBSupportMultipleScreenShots.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+var mongoose = require('mongoose'),
+    issue = require('./models/issue.js');
+
+var url = 'mongodb://localhost/sws';
+var db = mongoose.connection;
+mongoose.connect(url);
+
+issue.find().snapshot().forEach(
+  function (x) {
```
Comment (drkno) [inline updateDBSupportMultipleScreenShots.js]: x is not an amazing name

File: models/issue.js [hunk 5]
```diff
@@ -14,8 +14,8 @@ var issueSchema = new mongoose.Schema({
...
 	misc: String
 });
 
-module.exports = mongoose.model('issue', issueSchema);
\ No newline at end of file
+module.exports = mongoose.model('issue', issueSchema);
```
Comment (drkno) [inline models/issue.js:21]: what is actually different here?

Comment (Awarua-) [inline models/issue.js:21]: No newline at end of file I think

File: models/issue.js (hunk 1 above)
Comment (Awarua-) [inline models/issue.js:17]: Noted

**Comment by drkno** [comment]
Cannot approve without write access

**Pull request merged**